js/web/lib/wasm/jsep/webgpu/ops/unary-op.ts

```diff
@@ -259,7 +259,8 @@ export const tan = (context: ComputeContext): void => {
 };
 
 export const tanh = (context: ComputeContext): void => {
```

Comment:
should put a comment here explaining the situation and add a TODO item to remove it when the corresponding issue is fixed and implemented in browser.

Reply:
Today WGSL WG has [initial discussion](https://docs.google.com/document/d/1aSdSJoG0cZWSiOLa3YQrIZU5kM8eGA3tU2pUn0ErzBs/edit#heading=h.7t36p7inyd8j) on `tanh` in WGSL spec, they generally agree on making spec more strict to prevent NaN for large input, but they are not feeling urgent.
Maybe we can try to land some work around in certain browsers, but behavior might vary as long as spec allow indeterminate result.